src/components/TextWithTooltip.tsx

```diff
@@ -43,9 +45,9 @@ export const TextWithTooltip = ({
           <SvgIcon
             sx={{
               fontSize: iconSize,
-              color: open ? 'info.main' : 'text.muted',
+              color: color ? color : open ? 'info.main' : 'text.muted',
               borderRadius: '50%',
-              '&:hover': { color: 'info.main' },
+              '&:hover': { color: color || 'info.main' },
             }}
           >
             {icon || <InformationCircleIcon />}
```

Comment:
Allows the styling of color on the tooltip icon, i.e. for red/yellow exclamation tooltips.